naturblick/views/ObservationListView.swift

```diff
@@ -112,6 +181,12 @@ struct ObservationListView: HostedView {
                         errorHandler.handle(error)
                     }
                 }
+                .onChange(of: model.editMode) { _ in
+                    reinitNav()
+                }
+                .onChange(of: model.selectedItems) { _ in
```

Comment:
I think you should only trigger the reinit when selectedItems size changes between 0 and >0. Now the buttons flicker every time I select an item.